UPDATING.md

```diff
@@ -16,292 +16,300 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
+
 # Updating Superset
 
 This file documents any backwards-incompatible changes in Superset and
 assists people when migrating to a new version.
 
 ## Next
 
-* NOTICE: config flag ENABLE_REACT_CRUD_VIEWS has been set to `True` by default, set to `False` if
-  you prefer to vintage look and feel :)
+- [11509](https://github.com/apache/incubator-superset/pull/11509): Config value `TABLE_NAMES_CACHE_CONFIG` has been renamed to `DATA_CACHE_CONFIG`, which will now also hold query results cache from connected datasources, in addition to the table names.
```

Comment:
Probably worth calling out that if the `TABLE_NAMES_CACHE_CONFIG` uses a different cache then any existing cached result sets will be blown away.

Reply:
Added some text about cache warmup.